extensions/hibernate-search-orm-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/orm/elasticsearch/runtime/HibernateSearchElasticsearchBuildTimeConfigPersistenceUnit.java

```diff
@@ -244,8 +228,8 @@ public static class CoordinationConfig {
          *
          * @asciidoclet
          */
-        @ConfigItem(defaultValue = "none")
-        public Optional<String> strategy;
+        @WithDefault("none")
+        Optional<String> strategy();
```

Comment:
Hey, and optional with a default value looks odd... I know it's not new, but still. Shouldn't this be `@ConfigDocDefault` instead?

Reply:
Agreed that it's odd. I wasn't sure how to change it so kept it as is.